cmd/queue/main.go

```diff
@@ -197,7 +202,23 @@ func proxyHandler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, tracingEn
 	}
 }
 
-func knativeProbeHandler(healthState *health.State, prober func() bool, isAggressive bool, tracingEnabled bool, next http.Handler) http.HandlerFunc {
+func preferPodForScaledown(logger *zap.SugaredLogger, downwardAPILabelsPath string) bool {
```

Comment:
I think this code should short-circuit and always return `false` if the `downwardAPILabelsPath` is empty? Otherwise we'll have error logs if this feature is not activated.